telco-hub/configuration/reference-crs-kube-compare/optional/odf-internal/odfSubscription.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: odf-operator
+  namespace: openshift-storage
+spec:
+  channel: "stable-4.18"
```

Comment:
These all need to be 4.19 based, but that can be done at the same time as the actual source-crs.

Reply:
Agree. It's better to update the operator channels along with the templates in a different PR.